contracts/token/psp22/stub.rs

```diff
@@ -130,3 +152,76 @@ pub mod psp22receiver {
         }
     }
 }
+
+#[ink_lang::contract(compile_as_dependency = true)]
+pub mod psp22mintable {
+
+    #[derive(Default)]
+    #[ink(storage)]
+    pub struct PSP22Mintable {}
+
+    #[ink(namespace = "PSP22Mintable")]
+    impl PSP22Mintable {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            unimplemented!()
+        }
+
+        #[ink(message)]
+        pub fn mint(&mut self, amount: Balance) {
+            unimplemented!()
+        }
+    }
+}
+
+#[ink_lang::contract(compile_as_dependency = true)]
+pub mod psp22burnable {
+
+    #[derive(Default)]
+    #[ink(storage)]
+    pub struct PSP22Burnable {}
+
+    #[ink(namespace = "PSP22Burnable")]
+    impl PSP22Burnable {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            unimplemented!()
+        }
+
+        #[ink(message)]
+        pub fn burn(&mut self, amount: Balance) {
+            unimplemented!()
+        }
+
+        #[ink(message)]
+        pub fn burn_from(&mut self, account: AccountId, amount: Balance) {
+            unimplemented!()
+        }
+    }
+}
+
+#[ink_lang::contract(compile_as_dependency = true)]
+pub mod psp22wrapper {
+
+    #[derive(Default)]
+    #[ink(storage)]
+    pub struct PSP22Burnable {}
+
+    #[ink(namespace = "PSP22Burnable")]
+    impl PSP22Burnable {
```

Comment:
You forgot to change the name and missed some methods=)

Reply:
Names are updated now